common/persistence/sql/sqlplugin/postgresql/session/session.go

```diff
@@ -38,7 +39,8 @@ import (
 )
 
 const (
-	dsnFmt = "postgres://%v:%v@%v/%v?%v"
+	dsnFmt     = "postgres://%v:%v@%v/%v?%v"
+	driverName = "postgres"
```

Comment:
What is this used for when a connector is provided?

Reply:
According to the docs:
> The driverName of the original database is required for named query support.

You need to pass along the actual name of the driver here as we support both pq and pgx now

Reply:
Thanks for the review. I've made the changes here https://github.com/temporalio/temporal/pull/5265/commits/0476db1d69be30e2ce6c53487860644c78619cae